charts/kubernetes-agent/templates/tentacle-deployment.yaml

```diff
@@ -102,10 +102,16 @@ spec:
             {{- end }}
             - name: "OCTOPUS__K8STENTACLE__PODRESOURCEJSON"
               value: {{ .Values.scriptPods.resources | toJson | quote }}
+            - name: "OCTOPUS__K8STENTACLE__PODAFFINITYJSON"
+              value: {{ .Values.scriptPods.affinity | toJson | quote }}
```

Comment:
If someone overrides the affinity to something empty, should we replace with the default or assume they made the change on purpose?

Reply:
🤔 Good question.

For backwards compatibility, I think we _need_ to have it set the default. Existing users won't have a value in the values.yaml, so we don't want to set the affinity empty in those cases.

Until we hear otherwise, I'm comfortable having it just fallback on the hardcoded defaults